pkg/cmd/prune-backups/prune.go

```diff
@@ -33,10 +33,14 @@ type PruneOpts struct {
 	RetentionType      string
 	MaxNumberOfBackups int
 	MaxSizeOfBackupsGb int
+	BackupPath         string
```

Comment:
not sure how this relates to the backup sidecar?

Reply:
so I need to give the backup path for the Pruner, it was inline on the code, and since we use a diff backup path, I had to override it here

also on the pruner I pass it as arg, and use the inlined path as default, and override it here